semperfiwebdesign/all-in-one-seo-pack
# Link welcome screen to release post

Issue #2759

## Proposed changes

Link welcome screen to the latest release post.

## Types of changes

- Improves existing functionality

## Checklist

- [] Travis-ci runs with no errors.

## Testing instructions

Go to wp-admin/index.php?page=aioseop-about and check the link to the release post.
